src/components/views/settings/KeyBackupPanel.js

```diff
@@ -250,13 +250,19 @@ export default class KeyBackupPanel extends React.PureComponent {
                 backupSigStatuses = _t("Backup is not signed by any of your devices");
             }
 
+            let trustedLocally;
+            if (this.state.backupSigStatus.trusted_locally) {
+                trustedLocally = _t("This backup is trusted because it has been restored on this device");
+            }
+
             return <div>
                 {_t("Backup version: ")}{this.state.backupInfo.version}<br />
                 {_t("Algorithm: ")}{this.state.backupInfo.algorithm}<br />
                 {clientBackupStatus}<br />
                 {uploadStatus}
-                <div>{backupSigStatuses}</div><br />
-                <br />
+                <div>{backupSigStatuses}</div>
+                <div>{trustedLocally}</div>
+                <br /><br />
```

Comment:
imo we should be using block elements rather than breaks - breaks have me concerned for when/if someone changes their zoom level and things don't align anymore, whereas blocks would ensure the layout at least remains somewhat sane.